paddle/majel/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cc_library(majel SRCS place.cc)
+cc_library(majel SRCS place.cc ddim.cc)
```

Comment:
done.
But There is no `dim.cc` file. `Dim` is only a part of `DDim`, it is not an independent module and only has a 'dim.h' file.  So maybe these lines should be
```
cc_library(place SRCS place.cc)
cc_library(ddim SRCS ddim.cc)
```